docscan/session/create/sdk_config_test.go

```diff
@@ -0,0 +1,29 @@
+package create
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+func ExampleSdkConfigBuilder_Build() {
+	notifications, err := NewSdkConfigBuilder().
+		WithAllowsCamera().
+		WithErrorUrl("https://errorurl.com").
+		WithFontColour("#ff0000").
+		WithLocale("fr_FR").
+		WithPresetIssuingCountry("USA").
+		WithPrimaryColour("#aa1111").
+		WithSecondaryColour("#bb2222").
+		WithSuccessUrl("https://successurl.com").
+		Build()
+
+	if err != nil {
+		fmt.Fprintf(os.Stdout, "error: %s", err.Error())
```

Comment:
Why not just `fmt.Printf(...) ` ?

Reply:
Ah, I thought Fprintf was needed to give a good comparison when tests failed, but just tried with Printf and all looks good! Have updated that instance, and will update other instances in a separate PR

Reply:
Fixed in e16625a